modules/util/create.py

```diff
@@ -103,9 +118,11 @@ def create_model_loader(
                 return PixArtAlphaFineTuneModelLoader()
             if model_type.is_stable_diffusion_3():
                 return StableDiffusion3FineTuneModelLoader()
+            return None
```

Comment:
Maybe add a single `return None` at the end of each function instead of adding one for each case

Reply:
Done!